CONTRIBUTING.md

```diff
@@ -194,3 +194,36 @@ Each source file needs to have this header:
     // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     // See the License for the specific language governing permissions and
     // limitations under the License.
+
+## Quality levels
+
+CMake targets can be developed according to different quality levels. For production usage in a specific environment (e.g. compiler version, compiler flags, etc.), please contact one of the maintainers beforehand. All requirements of a lower quality level are included in higher quality levels e.g. quality level 4 is included in quality level 3.
```

Comment:
What is meant with production usage in this context? We cannot provide any guarantees here even if compiler version and flags are provided

Reply:
The idea is to state, that just the code is not enough for production usage, but more documents are needed.

Reply:
Rephrase it